src/lib/registry/severity-registry.js

```diff
@@ -0,0 +1,47 @@
+const Severity = require('../severity');
+const EmptyRegistry = require('./empty-registry');
+const { throwIf } = require('../utils/type-checks');
+
+class SeverityRegistry extends EmptyRegistry {
+    constructor() {
+        super();
+    }
+
+    add(name, instance) {
+        throwIf.not.string(name, 'name');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
        throwIf.not.string(name, "name");
```